README.md

```diff
@@ -1,29 +1,37 @@
 # imdb_dashboard
 
+The IMDb dashboard is primarily targeted towards movie producers to present a consolidated crisp view of the average ratings and runtime for movies by genres and regions with interactive abilities to help them choose and plan their next movie.
+
 ## Dashboard description
 
-Our dashboard consists of one web page that shows 4 main reactive plots:
+Our dashboard consists of one web page that shows overall summary and 4 main reactive plots:
+
+- The high level totals
 
-- The top rated movies by genre
+These are a set of 4 numeric totals showing the total count of movies, the average runtime across all movies, the total number of actors across all movies, and the average rating of all movies based on the filters applied.
 
-This is a bar chart showing the genres on the x-axis and some kind of aggregation (e.g. count or average rating) on the y-axis. Each of the individual bars is also coloured by genre.
+- The proportion of movies by genre
+
+This is a box plot showing the genres on the x-axis and the count of movies on the y-axis. The box plots will be categorized by a categorical column, for e.g., top 3 regions.
 
 - How movie ratings vary over time
 
 This is a line chart showing the years on the x-axis and the mean rating per genre on the y-axis. The individual lines may be coloured by other features such as genre or region of production.
 
 - The top rated movies by actor
 
-This is a bubble graph showing the mean rating on the y-axis and the year on the x-axis. Each bubble represents an actor and the size of the bubble represents the number of movies that that actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
+This is a bubble graph where each bubble represents an actor and the size and color of the bubble represents the number of movies that that actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
```

Comment:
```suggestion
This is a bubble graph where each bubble represents an actor and the size and color of the bubble represents the number of movies that the actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
```

Reply:
Updated the description for bubble chart